src/ElfUtils/ElfFile.cpp

```diff
@@ -294,25 +294,32 @@ std::filesystem::path ElfFileImpl<ElfT>::GetFilePath() const {
 template <typename ElfT>
 ErrorMessageOr<LineInfo> orbit_elf_utils::ElfFileImpl<ElfT>::GetLineInfo(uint64_t address) {
   CHECK(has_debug_info_section_);
-  auto line_info_or_error = symbolizer_.symbolizeCode(
-      *object_file_, {address, llvm::object::SectionedAddress::UndefSection});
+  auto line_info_or_error = symbolizer_.symbolizeInlinedCode(
+      object_file_->getFileName(), {address, llvm::object::SectionedAddress::UndefSection});
   if (!line_info_or_error) {
     return ErrorMessage(absl::StrFormat(
         "Unable to get line number info for \"%s\", address=0x%x: %s", object_file_->getFileName(),
         address, llvm::toString(line_info_or_error.takeError())));
   }
 
   auto& symbolizer_line_info = line_info_or_error.get();
+  const uint32_t number_of_frames = symbolizer_line_info.getNumberOfFrames();
+
+  // Getting back zero frames means there was some kind of problem. We will return a error.
```

Comment:
interesting, if it is easy to reproduce maybe add a test for this use-case?

Reply:
Ah, this is a tricky one. I haven't yet figured out when this conditon occurs and when the other one below occurs. So, unfortunately it's not so easy to reproduce.